src/t_stream.c

```diff
@@ -2029,30 +2029,50 @@ void xackCommand(client *c) {
     if (ids != static_ids) zfree(ids);
 }
 
-/* XPENDING <key> <group> [<start> <stop> <count> [<consumer>]]
+/* XPENDING <key> <group> [<start> <stop> <count> [<consumer>] [IDLE <idle>]]
```

Comment:
Adding review comment for this, so it's clear it's not resolved yet:
the command syntax is problematic, we rather not rely on the argc in order to know if the last argument is IDLE or consumer name (two args or one), since that'll prevent us from adding more args in the future. (keep in mind that a consumer can be named "idle")
maybe the consumer name should be mandatory argument, which can be "" in which case we take all.